src/testmod/java/example/mod/client/plugin/SampleComplexPolygonOverlayFactory.java

```diff
@@ -1,137 +1,137 @@
-package example.mod.client.plugin;

```

Comment:
I'm not entirely sure what happened here, I suspect it was using CRLF line endings previously but has been switched to LF due to my Git config.